gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -222,6 +222,7 @@ public void onStart() {
     // TODO: Create startup flow and move this logic there.
     Resource<Project> activeProject = viewModel.getActiveProject().getValue();
     if (activeProject == null || !activeProject.isLoaded()) {
+      hideBottomSheet();
```

Comment:
In my most recent PR, we restore the last active project onStart(), so this would only be run the first time the app is opened.